HOWTO.md

```diff
@@ -0,0 +1,37 @@
+# How to use PEPC
+## Enabling PEPC
+### Chrome
+Use a version of Chrome that is version 121.0.6136 or higher. This might mean you need to download a 
+[canary](https://www.google.com/chrome/canary/), [dev](https://www.google.com/chrome/dev/) or 
+[beta](https://www.google.com/chrome/beta/) version of Chrome. You can check your current browser version
+by vising `chrome://version`.
+
+To enable the PEPC feature you need to start Chrome with the command line flag: `--enable-features=PermissionElement`.
```

Comment:
Let's mention that on macOS you need to fully close the Chrome app for the setting to take effect, which you can verify by going to `chrome://version`.